tasks/11_find-longest-word/find-longest-word.js

```diff
@@ -0,0 +1,4 @@
+export function findLongestWord(str) {
+  return str.split(' ')
+      .reduce((max, curr) => curr.length > max.length? curr : max, '').length;
```

Comment:
Adding comments or splitting this single line solution in the future would be nice in order to increase readability